src/Database/Esqueleto/PostgreSQL.hs

```diff
@@ -91,7 +91,7 @@ unsafeSqlAggregateFunction name mode args orderByClauses = ERaw Never $ \info ->
                 [] -> ""
                 (_:_) -> " "
         (argsTLB, argsVals) =
-            uncommas' $ map (\(ERaw _ f) -> f info) $ toArgList args
```

Comment:
Ugh, we even have the right warning on. I need to do a warnings-clean PR on the project soon.